evmap-derive/Cargo.toml

```diff
@@ -26,4 +26,4 @@ quote = "1.0.2"
 proc-macro2 = "1.0.9"
 
 [dev-dependencies]
-evmap = "9.0.0"
+evmap = { path = "../" }
```

Comment:
I think the general advice here is to give _both_ a version _and_ a path. That way `git` builds and patched builds continue to work.